src/client/js/main.jsx

```diff
@@ -1,14 +1,140 @@
-var Test = React.createClass({
-    render() {
-        return (
-            <div>{this.props.msg}</div>
-        );
-    }
+/*
+
+	Copied, pasted and edited code from react-router example. To refine later
+
+ */
+
+
+var { Route, DefaultRoute, RouteHandler, Link } = ReactRouter;
```

Comment:
yes it is es6 "destructuring" syntax: https://github.com/lukehoban/es6features#enhanced-object-literals